docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md

```diff
@@ -37,7 +37,7 @@ Create a Data Asset to define the data you want GX Cloud to access. Currently, t
 
     - **Account identifier**: Enter your Snowflake account or locator information. The locator value must include the geographical region. For example, `us-east-1`. To locate these values see [Account Identifiers](https://docs.snowflake.com/en/user-guide/admin-account-identifier).
 
-    - **Password**: Enter your Snowflake password.
+    - **Password**: Enter your Snowflake password. To improve data security, GX recommends using a Snowflake service account to connect to GX Cloud.
```

Comment:
See the comment below. "a Snowflake password" instead of "your Snowflake password."

Maybe "Enter the password corresponding to the Snowflake user with you created in ____"

Reply:
@wookasz this issue has been addressed